### DISCUSSION TITLE
133

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#828] [Comments]
http://jojoldu.tistory.com/133

**Comment by jojoldu** [comment]
작성자: seungdols

여러모로 세미나 후기 자세하게 남겨주셔서 감사합니다. ㅎㅎ

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

오! 안녕하세요 승돌님! ㅎㅎ 
댓글 감사합니다!
회사 잘 적응하고 계시죠? ㅎㅎ
제이펍 리뷰 후기때 뵙겠습니다^^

**Comment by codelabor** [comment]
행사 준비했던 신상재입니다. 어쩌다 검색되어 옛 추억이 소환되네요. 큰 포부를 가지고 벌인 자리였는데 지속하지 못해 죄송합니다. 어떻게든 도움이 되었다니 다행이고 다음엔 또 다른 모습으로 뵙게 되면 좋겠습니다. :)
